File: jswebsocket/UniRpc/ReturnMessage.ts
import { BaseMessage } from "./BaseMessage";

export function ReturnMessage(message: BaseMessage, value: any): BaseMessage {
    return {
        Id: message.Id,
        Service: message.Service,
        Method: message.Method,
        GenericArguments: message.GenericArguments,
        Payload: value,
        Success: true
    };
}

File: tests/transformers/python/websocket-lambda-service/ReturnMessage.py
from UniRpc.BaseMessage import BaseMessage

def ReturnMessage(message: BaseMessage, value: object) -> BaseMessage:
    return {
        'Id': message['Id'],
        'Service': message['Service'],
        'Method': message['Method'],
        'GenericArguments': message['GenericArguments'],
        'Payload': value,
        'Success': message['Success']
    }